### PR TITLE
Disable even more FileWatching tests

### PR DESCRIPTION
**Pull request by gbaraldi**
These tests are also unreliable so disable them too for now. See https://github.com/JuliaLang/julia/issues/46185

### CODE REVIEW

**Comment by KristofferC** [comment]
Seems to still fail in FileWatching on mac

File: stdlib/FileWatching/test/runtests.jl [hunk 1]
```diff
@@ -32,7 +35,9 @@ for i in 1:n
     if !fd_in_limits && Sys.islinux()
         run(`ls -la /proc/$(getpid())/fd`)
     end
-    @test fd_in_limits
+    if ismacos_arm
```
Comment (KristofferC) [inline stdlib/FileWatching/test/runtests.jl:38]: Should this be `!ismacos_arm`? Does this test really only work on macOS ARM?

Comment (gbaraldi) [reply stdlib/FileWatching/test/runtests.jl:38]: Yeah, that was a typo. :-/

**Review by KristofferC — APPROVED** [review]

File: stdlib/FileWatching/test/runtests.jl [hunk 2]
```diff
@@ -32,7 +34,9 @@ for i in 1:n
     if !fd_in_limits && Sys.islinux()
         run(`ls -la /proc/$(getpid())/fd`)
     end
-    @test fd_in_limits
+    if !ismacos_arm
+        @test fd_in_limits
+    end
```
[the remark this answers is not in the document]
Comment (gbaraldi) [reply stdlib/FileWatching/test/runtests.jl:39]: Those tests are probably never going to be reliable on macos anyhow, I guess setting them as broken is cleaner. Not sure

**Comment by Keno** [comment]
I'm not particularly happy with the x86 vs aarch64 difference. These are mac kernel differences, not architecture specific issues. Either we expect the macos system to have these invariants or not. It shouldn't be platform specific.

**Pull request merged**